naff/models/discord/channel.py

```diff
@@ -2280,6 +2357,74 @@ async def create_post(
         )
         return self._client.cache.place_channel_data(data)
 
+    async def fetch_posts(self) -> List["GuildForumPost"]:
```

Comment:
`fetch_posts` only returns active posts, although the name suggests all posts will be fetched. Fetching archived posts should either be made an argument, or default behavior IMO.

Whatever happens a method to fetch archived posts is missing

Reply:
Archived threads were intentionally left out from this method (and the get equiv) because using it would be very expensive. Im open to adding a tertiary method (async iterator like other objects get) but im very opposed to including it in this method

Reply:
Sure, that works!